presto-docs/src/main/sphinx/installation.rst

```diff
@@ -8,5 +8,6 @@ Installation
     installation/deployment
     installation/cli
     installation/jdbc
+    installation/jmx
```

Comment:
I moved this under "administration"

Reply:
ok .. that makes sense too. I had it here with deployment .. but administration as managing the deployed system is fine too ..